src/components/details-view/details-view.js

```diff
@@ -198,6 +199,12 @@ export default class DetailsView extends Component {
 
     let historyState = router.history.location.state;
 
+    let countFilters = (params) => {
+      let { q, filter = {} } = params;
+      let appliedFilters = [q, ...Object.values(filter)].filter(val => !!val);
```

Comment:
Little know trick here: instead of `.filter(val => !!val)`, we can just do `.filter(Boolean)` to filter out any falsey values. Either way works, I just wanted to share 😄